master/buildbot/data/builds.py

```diff
@@ -95,8 +126,17 @@ def get(self, resultSpec, kwargs):
             builderid=kwargs.get('builderid'),
             buildrequestid=kwargs.get('buildrequestid'),
             complete=complete)
-        defer.returnValue(
-            [(yield self.db2data(dbdict)) for dbdict in builds])
+        # returns properties' list
+        selected_props = resultSpec.popProperties()
```

Comment:
[nit] there's a bit of inconsistency between variable name here and the parameter name in `_injectProperties`